File: Pertemuan 5/TugasPertemuan5.java
import java.util.Scanner;

public class TugasPertemuan5 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.print("Masukkan nama Anda: ");
        String nama = scan.nextLine();
        
        System.out.print("Masukkan Prodi Anda: ");
        String prodi = scan.nextLine();

        System.out.print("Masukkan nilai IPK Anda: ");
        double nilai = scan.nextDouble();

        System.out.print("Masukkan lama studi Anda (dalam tahun): ");
        double studi = scan.nextDouble();

        System.out.println("Nama Anda adalah " + nama);
        System.out.println("Prodi Anda Adalah " + prodi);
        if((nilai >= 3.51 && nilai <= 4.0) && (studi == 4)) {
            System.out.println("Selamat! Anda lulus dengan predikat Summa Cumlaude.");
        } else if((nilai >= 3.51 && nilai <= 4.0) && (studi > 4)) {
            System.out.println("Selamat! Anda lulus dengan predikat Cumlaude.");
        } else if((nilai >= 3.01 && nilai < 3.51) && (studi >= 4)) {
            System.out.println("Selamat! Anda lulus dengan predikat sangat memuaskan.");
        } else if((nilai >= 2.76 && nilai < 3.01) && (studi >= 4)) {
            System.out.println("Selama! Anda lulus dengan predikat memuaskan.");
        } else if((nilai >= 2.00 && nilai < 2.76) && (studi >= 4)) {
            System.out.println("Selamat! Anda lulus dengan predikat cukup.");
        } else if((nilai >= 0 && nilai < 2.00) && (studi >= 4)) {
            System.out.println("Mohon maaf! Anda tidak lulus.");
        } else {
            System.out.println("Nilai tidak valid!");
        } 
    }
}

File: Pertemuan 5/tugaspertemuan5.py
nama = input("Masukkan nama Anda: ")
prodi = input("Masukkan prodi Anda: ")
nilai = input("Masukkan nilai IPK Anda: ")
studi = input("Masukkan lama studi Anda (dalam tahun): ")

nilai = float(nilai)
studi = float(studi)

print("Nama Anda adalah " + nama)
print("Prodi Anda adalah " + prodi)
if (nilai >= 3.51 and nilai <= 4.0) and (studi == 4) :
    print("Selamat! Anda lulus dengan predikat Summa Cumlaude.")
elif (nilai >= 3.51 and nilai <= 4.0) and (studi > 4) :
    print("Selamat! Anda lulus dengan predikat Cumlaude.")
elif (nilai >= 3.01 and nilai < 3.51) and (studi >= 4) :
    print("Selamat! Anda lulus dengan predikat sangat memuaskan.")
elif (nilai >= 2.76 and nilai < 3.01) and (studi >= 4) :
    print("Selama! Anda lulus dengan predikat memuaskan.")
elif (nilai >= 2.00 and nilai < 2.76) and (studi >= 4) :
    print("Selamat! Anda lulus dengan predikat cukup.")
elif (nilai >= 0 and nilai < 2.00) and (studi >= 4) :
    print("Mohon maaf! Anda tidak lulus.")
else :
    print("Nilai tidak valid!")